DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy2.recipe.json

```diff
@@ -115,6 +115,59 @@
         {
             "name": "ContentDefinition",
             "ContentTypes": [
+                {
+                    "Name": "PersonalitySkill",
+                    "DisplayName": "Personality Skill",
+                    "Settings": {
+                        "ContentTypeSettings": {
+                            "Creatable": true,
+                            "Listable": true,
+                            "Draftable": true,
+                            "Versionable": true,
+                            "Securable": true
+                        },
+                        "FullTextAspectSettings": {}
+                    },
+                    "ContentTypePartDefinitionRecords": [
+                        {
+                            "PartName": "PersonalitySkill",
+                            "Name": "PersonalitySkill",
+                            "Settings": {
+                                "ContentTypePartSettings": {
+                                    "Position": "1"
+                                }
+                            }
+                        },
+                        {
+                            "PartName": "GraphSyncPart",
+                            "Name": "GraphSyncPart",
+                            "Settings": {
+                                "ContentTypePartSettings": {
+                                    "Position": "2"
+                                },
+                                "GraphSyncPartSettings": {
+                                    "NodeNameTransform": "ContentType",
+                                    "PropertyNameTransform": "Value",
+                                    "IdPropertyName": "uri",
+                                    "GenerateIdPropertyValue": "$\"<<contentapiprefix>>/{ContentType}/{Value}\".ToLowerInvariant()",
+                                    "DisplayId": true
+                                },
+                                "ContentIndexSettings": {
+                                    "Template": "undefined"
```

Comment:
Most types don't have this `undefined` (only `PersonalityTrait`).

What is it and do we need it?

Reply:
Probably don't - it's what an OC export spat out. I'll change it now